src/task3.java

```diff
@@ -0,0 +1,13 @@
+public class task3 {
+    public static void main(String...args) {
+        double radius = 23;
+        double area = Math.PI * radius * radius;
+        double line = 2 * Math.PI * radius;
+            System.out.println("Area=" + area + ";" + "Line=" + line);
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"